### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by mr-fixit**
for those who RTFM

### CODE REVIEW

File: clangFormat/README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+## Code Style
+The clang format pre-commit hook will execute clang-format on your code after you commit and inform you if there were any style violations.  All files can be found here: [Code Style](https://github.com/Raizlabs/Raizlabs-Cocoa-Style/tree/master/clangFormat)
```
Comment (ZevEisenberg) [inline clangFormat/README.md]: #req double space after `violations.`

File: clangFormat/README.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+ - Download the private clang-format binary, copy it into /usr/local/bin/ and chmod +x it.
```
Comment (ZevEisenberg) [inline clangFormat/README.md]: #nth explain why we are using a private binary (I think it's because of @KingOfBrian's fix, right?).

Comment (mr-fixit) [inline clangFormat/README.md]: I don't know why we have a custom binary, I assumed it was simply to keep from having to build it.  I've asked NickB and will put his answer in the README.

**Comment by mr-fixit** [comment]
@jatraiz would you like to merge this PR?

**Comment by ZevEisenberg** [comment]
#req add a section on removing the quarantine xattr bit. Come find me today and I’ll show you the incantation.

**Comment by ZevEisenberg** [comment]
@mr-fixit to elaborate:

```sh
xattr -d com.apple.quarantine /path/to/clang-format
```

**Comment by mr-fixit** [comment]
@ZevEisenberg how would that bit get set?  by installing in some other way than 'git clone'?

**Comment by ZevEisenberg** [comment]
@mr-fixit if you download the clang-format binary directly via Safari instead of cloning the repository. See for yourself:

```sh
ls -la@ ~/Downloads
```

**Comment by mr-fixit** [comment]
@ZevEisenberg @jatraiz Zev, thanks for the tip!

**Pull request merged**